tools/add_triage_signature.py

```diff
@@ -412,45 +478,58 @@ def get_logs_url_from_issue(issue):
 def get_all_triage_tickets(jclient, only_recent=False):
     recent_filter = "" if not only_recent else 'and created >= -31d'
     query = 'project = MGMT AND component = "Assisted-installer Triage" {}'.format(recent_filter)
-    idx = 0
-    block_size = 100
-    issues = []
-    while True:
-        i = jclient.search_issues(query, maxResults=block_size, startAt=idx)
-        if len(i) == 0:
-            break
-        issues.extend(i)
-        idx += block_size
+
+    return jclient.search_issues(query, maxResults=None)
+
+
+def get_issues(jclient, issue, recent_issues):
+    if issue is None:
+        logger.info(f"Fetching {['all', 'recent'][recent_issues]} issues, this may take a while")
```

Comment:
This isn't clear. Why not `{'recent' if recent_issues else 'all'}`?
In addition, I would change the variable name to be clear it's a boolean i.e. `only_recent_issues`

Reply:
> Why
Because it's shorter and in my opinion more readable, will change it to ternary and `only_*`